### ISSUE TITLE
System.IO.FileLoadException: Could not load file or assembly 'System.Net.Http.Extensions, Version=1.5.0.0

### ISSUE THREAD

**Issue by id0001** [object/Simple.OData.Client#424]
Hi,

I'm building a Dynamics CRM 365 connector for Microsoft Identity Manager and I'm presented with the error in the title when I try to run the connector from MIM. 
I'm using version 4.28.0 of the NuGet package on .NET version 4.6

Because MIM runs connectors as plugins, the bindingredirection in app.config will not work unless I add this to the app.config of the server executable. This works but is not a feasible solution in production.

I'm not sure if this is an actual bug in Simple.Odata.Client, but any help would be appreciated in solving this problem.

The stacktrace from Eventviewer: https://pastebin.com/eZeW5D4k

**Comment by object** [comment]
For your information: I just pushed the version 5.0.0-beta2 that supports .NET Standard 2.0 and updated to use ODataLib7. Please check it out.

**Comment by acesyde** [comment]
@object Hi, same error on 5.0.0-beta2 on .Net Core Console Application 

> One or more errors occurred. (Could not load file or assembly 'System.Net.Http.Extensions, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

**Comment by dmeierotto** [comment]
Same issue here, I am using the 5.0.0-beta2 package from an asp.net core application. 

> One or more errors occurred. (Could not load file or assembly 'System.Net.Http.Extensions, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.)

**Comment by ozzioma** [comment]
Having same issues here with the -Version 5.0.0-beta1 and -Version 5.0.0-beta2 packages.
Using the lib from a Xamarin Forms app, 2.5 with a .Net Standard 2.0 PCL.
Any workarounds to this? 
Spent half the week tracing this exception.

**Comment by wickaaro** [comment]
Same issue on .NET Core 2.0 web application.

**Comment by drwatson1** [comment]
I have the same issue with .Net 4.5 project and version 5.0.0-beta2. But I found that the error occurs only if I use Simple.OData.V4.Client. When I switched to Simple.OData.Client than the error has disappeared. I think this can be a workaround for this issue.

**Comment by ericleigh007** [comment]
I have the same issue.
I'm using Azure functions to move some OData from D365 Sales to a sharepoint list.
The error with the Client.V4 was 
Exception while executing function: QueueCommandFunctionsOne or more errors occurred.Could not load file or assembly 'System.Net.Http.Extensions, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

**Comment by cherici-simone** [comment]
Hello, I try the workaround (Use Client instead of Client.V4) but I receive the same error: 

> System.IO.FileNotFoundException: Could not load assembly 'System.Net.Http.Extensions, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Perhaps it doesn't exist in the Mono for Android profile?

**Comment by S1r-Lanzelot** [comment]
@ODataSyd https://github.com/object/Simple.OData.Client/issues/454 and https://github.com/object/Simple.OData.Client/issues/460

**Comment by object** [comment]
This should work fine in version 5.0.0.1.

**Comment by object** [comment]
@acesyde  @id0001 @dmeierotto @ozzioma @wickaaro @drwatson1 @ericleigh007 @cherici-simone @MisterJimson Please try 5.0.0.1 Should work now.

**Comment by MisterJimson** [comment]
Thanks. I’ll try tomorrow and report back.

**Comment by object** [comment]
Yes, this is not related to the original issue and I already received feedback from others. Will investigate.
